### DISCUSSION TITLE
SSH Directory/File

### DISCUSSION THREAD

**Discussion by upsidedly** [thebaselab/codeapp#950] [Q&A]
How do I open singular files or open directories in a ssh'd server in the app?

**Comment by bummoblizard** [comment]
https://code.thebaselab.com/guides/connecting-to-a-remote-server-ssh-ftp

**Reply by upsidedly** [reply]
I figured it out, thanks. How would I paste into the terminal though

**Comment by bummoblizard** [comment]
You can use Command-P or the paste button on your keyboard.

「Upsided ***@***.***>」在 2023年9月7日 週四，20:52 寫道：
